pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -28,7 +36,100 @@ type Adapter struct {
 }
 
 // Start updates the component if a matching component exists or creates one if it doesn't exist
-func (a Adapter) Start() (err error) {
+// Once the component has started, it will sync the source code to it.
+func (a Adapter) Start(path string, out io.Writer, ignoredFiles []string, forceBuild bool, globExps []string, show bool) (err error) {
```

Comment:
`out` and `show` don't appear to be used, can we remove them?

Reply:
They are used, in the call to `pushLocal`

Reply:
Done